third-party-programs.txt

```diff
@@ -0,0 +1,570 @@
+                            University Research and Technology Corporation. All
+                            rights reserved.
+    Copyright (c) 2004-2005 The University of Tennessee and The University of
+                            Tennessee Research Foundation. All rights reserved.
+    Copyright (c) 2004-2005 High Performance Computing Center Stuttgart,
+                            University of Stuttgart. All rights reserved.
+    Copyright (c) 2004-2005 The Regents of the University of California. All
+                            rights reserved.
+    Copyright (c) 2009      CNRS
+    Copyright (c) 2009-2016 Inria. All rights reserved.
+    Copyright (c) 2009-2015 Université Bordeaux
+    Copyright (c) 2009-2015 Cisco Systems, Inc. All rights reserved.
+    Copyright (c) 2009-2012 Oracle and/or its affiliates. All rights reserved.
+    Copyright (c) 2010      IBM
+    Copyright (c) 2010      Jirka Hladky
+    Copyright (c) 2012      Aleksej Saushev, The NetBSD Foundation
+    Copyright (c) 2012      Blue Brain Project, EPFL. All rights reserved.
+    Copyright (c) 2013-2014 University of Wisconsin-La Crosse. All rights
+                            reserved.
+    Copyright (c) 2015      Research Organization for Information Science and
+                            Technology (RIST). All rights reserved.
+    Copyright (c) 2015-2016 Intel, Inc. All rights reserved.
+
+    See COPYING in top-level directory.
+
+    Redistribution and use in source and binary forms, with or without
+    modification, are permitted provided that the following conditions
+    are met:
+    1. Redistributions of source code must retain the above copyright
+       notice, this list of conditions and the following disclaimer.
+    2. Redistributions in binary form must reproduce the above copyright
+       notice, this list of conditions and the following disclaimer in the
+       documentation and/or other materials provided with the distribution.
+    3. The name of the author may not be used to endorse or promote products
+       derived from this software without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED BY THE AUTHOR ``AS IS'' AND ANY EXPRESS OR
+    IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+    OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
+    IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT
+    NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+    DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+    THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+    (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
+    THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+_______________________________________________________________________________
+
+3. jemalloc
```

Comment:
Do we copy, or distribute jemalloc? if not we don't need it here

Reply:
if we link with jemalloc statically, then yes